packages/webpack/get-config.js

```diff
@@ -35,13 +34,9 @@ module.exports = async function getConfig (limitConfig, check, output) {
       path: output
     },
     optimization: {
-      concatenateModules: !check.disableModuleConcatenation
-    },
-    resolve: {
-      plugins: [PnpWebpackPlugin]
-    },
-    resolveLoader: {
-      plugins: [PnpWebpackPlugin.moduleLoader(module)]
+      concatenateModules: !check.disableModuleConcatenation,
+      minimize: true,
+      minimizer: ['...', new CssMinimizerPlugin()]
```

Comment:
I don't fully understand the `'...'`, but [the webpack docs say it's the way to do things for webpack v5](https://webpack.js.org/plugins/css-minimizer-webpack-plugin/#getting-started). I think it means allow the default ones, as well as the custom one being provided.